radix-engine-tests/tests/wasm_validation.rs

```diff
@@ -26,3 +27,51 @@ fn test_large_memory() {
         result
     );
 }
+
```

Comment:
Can we add atleast one test which captures the `PackageError::InvalidName` error

Reply:
Done